lambda_manage.py

```diff
@@ -0,0 +1,19 @@
+import os
+import logging
+
+from flask_migrate import upgrade
+
+from api.app import create_app
+from api.conf import AWSProductionConfig
+
+'''Provides management tasks which can be invoked using AWS Lambda'''
```

Comment:
Would be better to put it on top of the module as single docstring
https://stackoverflow.com/a/708674